syntax="proto3";

package atom.proto;

message Vector3 {
	float x=1;
	float y=2;
	float z=3;
}
